sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -19,9 +19,14 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
                             target=None, response_method='auto', n_cols=3,
                             grid_resolution=100, percentiles=(0.05, 0.95),
                             method='auto', n_jobs=None, verbose=0, fig=None,
-                            line_kw=None, contour_kw=None, ax=None):
-    """Partial dependence plots.
-
+                            line_kw=None, contour_kw=None, ax=None,
```

Comment:
I think that we should introduce a parameter `subsample` accepting both int and float to plot a subset of the ICEs. It will avoid a user to have to take a subset sample (which will also influence the computation of the averaged).

Reply:
Makes sense. I will add that.

Reply:
Added in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/f6f2f3cf73d029f3b48c1f8617dcd3dba1d95f9c